src/map/packets.hpp

```diff
@@ -760,6 +760,13 @@ struct PACKET_ZC_OPEN_EDITDLGSTR {
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_OPEN_EDITDLGSTR, 0x1d4)
 
+struct PACKET_ZC_DELETE_RELATED_GUILD {
+	int16 packetType;
+	int relationGuildId;
```

Comment:
```suggestion
	uint32 relationGuildId;
```

Reply:
i followed emulator decl
![image](https://github.com/user-attachments/assets/ea24f76f-455a-404d-8ea4-5373384f4e1f)

Reply:
should be the other way around, we always need to send the correct type for the packets